script.js

```diff
@@ -646,4 +646,30 @@ form.addEventListener('submit',(event)=>{
        event.preventDefault();
   }
  
-})
\ No newline at end of file
+})
+
+const username =form.elements.username.value;
+const email = form.elements.email.value;
+const message=form.elements.message.value;
+
+const populateLocalStorage = ()=>{
+  const data = {};
+  data.username=username;
+  data.email= email;
+  data.message =message;
+  console.log('localstorage')
+  localStorage.setItem('data',JSON.stringify(data));
+} 
```

Comment:
- [ ] Having made a great effort working on the local storage project. I notice my data is not populated to the local storage and anytime I visit the page again my data initially inserted is not found.
 
Here is a picture of your project to my claim 👇 

![she-local](https://user-images.githubusercontent.com/78634841/209238798-ef00d042-3b35-4606-87d1-a88b3f736a8d.png)

Kindly note the following while working on the local storage 👇 
  - The form data for all inputs should be stored in a single object, and saved to a single local storage registry.
  e.g
```JS
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```
  - When the user changes the content of any input field in the form the data object is updated and it is saved to the local storage.
  - When the user loads the page, the code checks if there are any saved data in the local storage and if so the input fields are pre-filled with the data.

Kindly Fix this.

Reply:
Thanks. I applied your suggestion.